doc/cascadia/profiles.schema.json

```diff
@@ -2276,8 +2282,14 @@
           ]
         },
         "adjustIndistinguishableColors": {
-          "description": "When set to true, we will (when necessary) adjust the foreground color to make it more visible, based on the background color.",
-          "type": "boolean"
+          "default": "Never",
```

Comment:
i don't think i ever realized our schema has two copies of all these objects ... and i can't figure out why

Reply:
Because they are both in `AppearanceConfig` and in `Profile` (this is annoying tbh)

Reply:
```suggestion
          "default": "never",
```